### PR TITLE
ENCD-4107: added experiment_class to experiment schema

### PR DESCRIPTION
**Pull request by weiweizhong2018**

### CODE REVIEW

File: src/encoded/schemas/experiment.json [hunk 1]
```diff
@@ -232,7 +232,15 @@
...
-            ]
+            ],
+            "experiment_class": {
+                "title": "Experiment class",
+                "description": "The categorization of the experiment.",
+                "type": "string",
+                "enum" : [
+                    "ground level annotation",
+                    "functional characterization"
+                ]
```
Comment (caseylitton) [inline src/encoded/schemas/experiment.json]: Looks like there is a '}' missing here.

**Comment by weiweizhong2018** [comment]
Thanks @caseylitton ! Now it works. I will change the Jira ticket to code review ready.

File: src/encoded/schemas/experiment.json [hunk 2]
```diff
@@ -234,6 +234,15 @@
                 "organoid"
             ]
         },
+        "experiment_class": {
+            "title": "Experiment class",
+            "description": "The categorization of the experiment.",
+            "type": "string",
+            "enum" : [
+                "ground level annotation",
```
Comment (estherc) [inline src/encoded/schemas/experiment.json]: I'm not sold on this enum value though I don't have a viable alternate to offer at the moment.

**Comment by estherc** [comment]
Can you add this property in some of the example experiment test inserts please?

File: src/encoded/schemas/changelogs/experiment.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for experiment.json
 
+### Schema version 21
+
+* Added *experiment_classification* property with enum "functional genomics assay", "functional characterization assay"
```
Comment (estherc) [inline src/encoded/schemas/changelogs/experiment.md]: How about "Added **required** property *experiment_classification* with enums "functional genomics assay" and "functional characterization assay"

File: src/encoded/upgrade/dataset.py [hunk 4]
```diff
@@ -384,3 +384,10 @@ def dataset_19_20(value, system):
 def dataset_20_21(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-3889
     return
+
+
+@upgrade_step('experiment', '20', '21')
+def dataset_21_22(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-4107
+    value['experiment_classification']=["functional genomics assay"]
+    return
```
Comment (estherc) [inline src/encoded/upgrade/dataset.py]: Should add a newline at the end of the file...

**Pull request merged**